### PR TITLE
Adding information about AliFone company's use of Airflow

### PR DESCRIPTION
**Pull request by oregisam**
---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: INTHEWILD.md [hunk 1]
```diff
@@ -523,3 +523,4 @@ Currently, **officially** using Airflow:
 1. [Zynga](https://www.zynga.com)
 1. [Ørsted](https://orsted.com/en) [[@arjunanan6](https://github.com/arjunanan6)]
 1. [好大夫在线](https://www.haodf.com/) [[@leiguorui](https://github.com/leiguorui)]
+1. [AliFone](https://www.alifone.com.br/)]
```
Comment (pankajastro) [inline INTHEWILD.md]: this list is sorted can you please adjust it accordingly

Comment (oregisam) [reply INTHEWILD.md]: Thanks

File: INTHEWILD.md [hunk 2]
```diff
@@ -522,4 +523,4 @@ Currently, **officially** using Airflow:
 1. [Zymergen](https://www.zymergen.com/)
 1. [Zynga](https://www.zynga.com)
 1. [Ørsted](https://orsted.com/en) [[@arjunanan6](https://github.com/arjunanan6)]
-1. [好大夫在线](https://www.haodf.com/) [[@leiguorui](https://github.com/leiguorui)]
+1. [好大夫在线](https://www.haodf.com/) [[@leiguorui](https://github.com/leiguorui)]
```
Comment (pankajastro) [inline INTHEWILD.md:526]:
```suggestion
1. [好大夫在线](https://www.haodf.com/) [[@leiguorui](https://github.com/leiguorui)]

```

Comment (pankajastro) [inline INTHEWILD.md]:
```suggestion
1. [好大夫在线](https://www.haodf.com/) [[@leiguorui](https://github.com/leiguorui)]

```
looks like earlier there was a new line

**Review by pankajastro — APPROVED** [review]

**Review by Taragolis — APPROVED** [review]

**Pull request merged**